src/backend/src/common/constants/roles.constants.ts

```diff
@@ -0,0 +1,112 @@
+import { GovMinistriesEnum } from '../enums/gov-ministries.enum';
+import { RolesEnum } from '../enums/roles.enum';
+import { UserTypesEnum } from '../enums/users.enum';
+
+export interface IRoleInfo {
+  user: UserTypesEnum;
+  ministry: GovMinistriesEnum;
+}
+
+export const Roles: Record<RolesEnum, IRoleInfo> = {
```

Comment:
The changes mentioned above will, of course, have to be reflected in this file as well.